Dockerfile

```diff
@@ -13,9 +13,10 @@ ENV TERM linux
 
 # Airflow
 ARG AIRFLOW_VERSION=1.10.3
-ARG AIRFLOW_HOME=/usr/local/airflow
+ARG AIRFLOW_USER_HOME=/usr/local/airflow
```

Comment:
Why change the arg name - you can (and often do) have the same ARG and ENV name.

Reply:
In fact, Airflow user home (system) may be different than Airflow home dir(application). That's why i used two differents args.

Reply:
It's also useful to have the ENV AIRFLOW_HOME propagate into downstream Dockerfiles, which doesn't happen with an ARG.